crates/ruma-html/Cargo.toml

```diff
@@ -18,7 +18,7 @@ matrix = ["dep:ruma-common"]
 
 [dependencies]
 as_variant = { workspace = true }
-html5ever = "0.28.0"
+html5ever = "0.29.0"
```

Comment:
They released 2 versions with breaking changes in a row? :roll_eyes:

Reply:
Well this was a tiny breaking change. But would be nice if they had a changelog / release notes 😅